permissions/RemovableMedia.permission

```diff
@@ -0,0 +1,13 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key-description = permission-la-removable_media
+# x-sailjail-description = Removable media
+# x-sailjail-translation-key-long-description = permission-la-removable_media_description
+# x-sailjail-long-description = Use memory cards and USB sticks
+
+### Allow to listen/talk to udisks2
+# Used for mounting, formatting, locking only
```

Comment:
Perhaps this comment should be removed still.

Reply:
Oh yeah, that is a remnant that should be removed.

Reply:
Removed by editing the relevant commit and force pushing. Resolving.